shell/common/animator.h

```diff
@@ -75,9 +76,11 @@ class Animator final {
 
   void SetDimensionChangePending();
 
-  // Enqueue |trace_flow_id| into |trace_flow_ids_|.  The corresponding flow
-  // will be ended during the next |BeginFrame|.
-  void EnqueueTraceFlowId(uint64_t trace_flow_id);
+  // Enqueue |packet| into |pointer_data_| and |trace_flow_id| into
+  // |trace_flow_ids_|. Pointer delta counters will be updated and
+  // the corresponding flow will be ended during the next |BeginFrame|.
+  void EnqueueTraceData(const PointerDataPacket* packet,
```

Comment:
It looks like this should never be null, right (it looks like it gets unconditionally dereferenced)?  Should we make this a ref-to-const instead?

Reply:
Done